source/jormungandr/jormungandr/scenarios/tests/new_default_tests.py

```diff
@@ -343,48 +343,62 @@ def add_pt_sections(journey):
 def get_kraken_calls_test():
     for md in ["bike", "walking", "bss", "car", "ridesharing", "taxi"]:
         req = {"origin_mode": [md], "destination_mode": [md]}
-        assert get_kraken_calls(req) == {(md, md)}
+        assert get_kraken_calls(req) == {(md, md, "indifferent")}
 
-    req = {"origin_mode": ["bss", "walking"], "destination_mode": ["walking"]}
-    assert get_kraken_calls(req) == {("walking", "walking")}
+    req = {"origin_mode": ["bss", "walking"], "destination_mode": ["walking"], "direct_path": "none"}
+    assert get_kraken_calls(req) == {("walking", "walking", "none")}
 
-    req = {"origin_mode": ["bike", "walking"], "destination_mode": ["walking"]}
-    assert get_kraken_calls(req) == {("walking", "walking"), ("bike", "walking")}
+    req = {"origin_mode": ["bike", "walking"], "destination_mode": ["walking"], "direct_path": "only"}
+    assert get_kraken_calls(req) == {("walking", "walking", "only"), ("bike", "walking", "only")}
 
-    req = {"origin_mode": ["bike", "walking"], "destination_mode": ["bss"]}
-    assert get_kraken_calls(req) == {("bike", "bss")}
+    req = {"origin_mode": ["bike", "walking"], "destination_mode": ["bss"], "direct_path": "indifferent"}
+    assert get_kraken_calls(req) == {("bike", "bss", "indifferent")}
 
     req = {"origin_mode": ["bike", "car"], "destination_mode": ["bss"]}
-    assert get_kraken_calls(req) == {("bike", "bss"), ("car", "bss")}
+    assert get_kraken_calls(req) == {("bike", "bss", "indifferent"), ("car", "bss", "indifferent")}
 
     req = {"origin_mode": ["bss", "bike"], "destination_mode": ["bike"]}
-    assert get_kraken_calls(req) == {("bike", "bike")}
+    assert get_kraken_calls(req) == {("bike", "bike", "indifferent")}
 
     req = {"origin_mode": ["taxi", "bike"], "destination_mode": ["walking", "bss", "car", "ridesharing", "taxi"]}
     assert get_kraken_calls(req) == {
-        ('taxi', 'taxi'),
-        ('taxi', 'bss'),
-        ('bike', 'bss'),
-        ('bike', 'walking'),
-        ('bike', 'taxi'),
-        ('taxi', 'walking'),
-        ('taxi', 'ridesharing'),
+        ('taxi', 'taxi', "indifferent"),
+        ('taxi', 'bss', "indifferent"),
+        ('bike', 'bss', "indifferent"),
+        ('bike', 'walking', "indifferent"),
+        ('bike', 'taxi', "indifferent"),
+        ('taxi', 'walking', "indifferent"),
+        ('taxi', 'ridesharing', "indifferent"),
```

Comment:
Same, I think (bike, bss) and (bike, ridesharing) are missing originally here...